OWNERS_ALIASES

```diff
@@ -2,28 +2,10 @@
 
 aliases:
   code-reviewers:
-    - michaellevy101
-    - avishayt
-    - empovit
-    - eranco74
-    - ori-amizur
-    - tsorya
-    - yevgeny-shnaidman
-    - masayag
-    - carbonin
-    - rollandf
-    - danielerez
-    - ybettan
-    - slaviered
-    - asalkeld
+    - adriengentil
+    - rccrdpccl
```

Comment:
you can put yourselves as code-approvers as well, or at least yourself
at this point you know this infra better than I :)